rest-api/src/main/resources/paths/apps/v1_apps_appId_participants_userId_data.yml

```diff
@@ -0,0 +1,49 @@
+get:
+    operationId: getAllDataForAdminWorker
+    summary: Get list of identifiers for participant data
+    description: Endpoint for worker or admin to get a list of participant data identifiers.
+    tags:
+        - Participant Data
+        - _For Workers
+        - _For Admins
+    security:
+        - BridgeSecurity: []
+    parameters:
+        - $ref: ../../index.yml#/parameters/appId
+        - $ref: ../../index.yml#/parameters/userId
+        - $ref: ../../index.yml#/parameters/offsetKey
+        - $ref: ../../index.yml#/parameters/pageSize
+    responses:
+        200:
+            description: OK
+            schema:
+                $ref: ../../definitions/paged_resources/string.yml
```

Comment:
Does this mean that getAllDataForAdminWorker should return a ForwardCursorPagedResourceList<ParticipantData> instead of ForwardCursorPagedResourceList<String>? As it stands, both getAllDataForSelf() and getAllDataForAdminWorker() both return a list of identifiers, not a list of participant data.

Reply:
My mistake. You're right, this returns a list of identifiers, not a list of ParticipantData. So paged_resources/string is correct.